packages/tenant-process/test/api/getTenantVerifiedAttributeVerifiers.test.ts

```diff
@@ -0,0 +1,162 @@
+    },
+  ];
+
+  const serviceResponse = {
+    results: mockVerifiers,
+    totalCount: 1,
+  };
+
+  beforeEach(() => {
+    tenantService.getTenantVerifiedAttributeVerifiers = vi
+      .fn()
+      .mockResolvedValue(serviceResponse);
+  });
+
+  const authorizedRoles: AuthRole[] = [
+    authRole.M2M_ROLE,
+    authRole.M2M_ADMIN_ROLE,
+  ];
+
+  const unauthorizedRoles: AuthRole[] = [
+    authRole.ADMIN_ROLE,
+    authRole.API_ROLE,
+    authRole.SECURITY_ROLE,
+    authRole.SUPPORT_ROLE,
+    authRole.INTERNAL_ROLE,
+  ];
+
+  const makeRequest = async (
+    token: string,
+    requestTenantId: TenantId = tenantId,
+    requestAttributeId: AttributeId = attributeId,
+    queryParams = { limit, offset }
+  ) =>
+    request(api)
+      .get(
+        `/tenants/${requestTenantId}/attributes/verified/${requestAttributeId}/verifiers`
+      )
+      .query(queryParams)
+      .set("Authorization", `Bearer ${token}`)
+      .set("X-Correlation-Id", generateId());
+
+  it.each(authorizedRoles)(
+    "Should return 200 for user with role %s",
+    async (role) => {
+      const token = generateToken(role);
+      const res = await makeRequest(token);
+      expect(res.status).toBe(200);
+      expect(res.body).toEqual(serviceResponse);
+      expect(
+        tenantService.getTenantVerifiedAttributeVerifiers
+      ).toHaveBeenCalledWith(
+        tenantId,
+        attributeId,
+        { offset, limit },
+        expect.objectContaining({
+          logger: expect.any(Object),
+        })
+      );
+    }
+  );
+
+  it.each(unauthorizedRoles)(
+    "Should return 403 for user with role %s",
+    async (role) => {
+      const token = generateToken(role);
+      const res = await makeRequest(token);
+      expect(res.status).toBe(403);
+    }
+  );
+
+  it("Should return 200 with empty results when no verifiers exist", async () => {
+    const emptyResponse = { results: [], totalCount: 0 };
+    tenantService.getTenantVerifiedAttributeVerifiers = vi
+      .fn()
+      .mockResolvedValue(emptyResponse);
+
+    const token = generateToken(authRole.M2M_ROLE);
+    const res = await makeRequest(token);
+    expect(res.status).toBe(200);
+    expect(res.body).toEqual(emptyResponse);
+  });
+
+  it("Should handle pagination parameters correctly", async () => {
+    const customOffset = 5;
+    const customLimit = 20;
+    const token = generateToken(authRole.M2M_ROLE);
+
+    await makeRequest(token, tenantId, attributeId, {
+      offset: customOffset,
+      limit: customLimit,
+    });
+
+    expect(
+      tenantService.getTenantVerifiedAttributeVerifiers
+    ).toHaveBeenCalledWith(
+      tenantId,
+      attributeId,
+      { offset: customOffset, limit: customLimit },
+      expect.objectContaining({
+        logger: expect.any(Object),
+      })
+    );
+  });
+
+  it("Should return 404 when tenant does not exist", async () => {
+    const { tenantNotFound } = await import("../../src/model/domain/errors.js");
```

Comment:
No need for import here, you can do a normal import - I think this test is failing because you miss the error mapper that maps tenantNotFound to 404

```suggestion
```